config/config.exs

```diff
@@ -38,4 +38,30 @@ for config_file <- Path.wildcard("config/{deps,dotcom}/*.exs") do
   import_config("../#{config_file}")
 end
 
+config :mbta_metro, :map, %{
+  center: [-71.0589, 42.3601],
+  style: %{
+    "version" => 8,
+    "sources" => %{
+      "raster-tiles" => %{
+        "type" => "raster",
+        "tiles" => ["https://mbta-map-tiles-dev.s3.amazonaws.com/osm_tiles/{z}/{x}/{y}.png"],
```

Comment:
Running locally, I'm seeing a blank map. Are there additional setup steps I need to follow to see the map tiles?

Reply:
https://mybrowseraddon.com/access-control-allow-origin.html